### DISCUSSION TITLE
I can't connect to the admin site in production

### DISCUSSION THREAD

**Discussion by acyment** [cookiecutter/cookiecutter-django#5077] [Q&A] [answered]
Hi all. I have a site created using cookiecutter-django w/Docker, webpack, DRF, whitenoise, celery & sentry. I can access the website in production without any problem. Both the website and the admin work great locally. The problem is when I try to access the admin site in production. When I try to access it I get the main site, but the page title changes to "Page not found".
I am not sure whether this is a bug, my fault or what. I have checked the logs of django, nginx and traefik, but found no clue at all.
After a whole day trying to make Phind 70b, ChatGPT4 or Claude Plus give me a hint with this, I hereby ask you guys for help. 
Thanks!
Alan

**Comment by browniebroke** [comment, marked as answer]
Did you set a specific value for the admin URL? By default it's a random value set by an environment variable:

https://github.com/cookiecutter/cookiecutter-django/blob/master/%7B%7Bcookiecutter.project_slug%7D%7D%2Fconfig%2Fsettings%2Fproduction.py#L228-L231

**Comment by acyment** [comment]
I can't believe I didn't read production.py on this one. It never occurred to me adding a second layer of security by having a secret admin url. Now that I know about it, I agree it makes sense. Thanks!
